src/components/Todo/Todo.js

```diff
@@ -14,13 +14,13 @@ const Todo = ({
   status,
   onStatusChange,
   onDelete,
-  onUpdateText,
+  onUpdateTitle,
   onDetailsTodoClick,
   children,
   ...props
 }) => {
   const [isMouseOver, setIsMouseOver] = useState(false)
-  const [todoText, setTodoText] = useState(children)
+  const [todoTitle, setTodoTitle] = useState(children)
```

Comment:
Como estamos dentro de `Todo`, `todoTitle` es un poco redundante (a menos que tuviésemos otro título dentro del mismo, que por ahora no es el caso), por lo que podemos directamente poner `title`

```suggestion
  const [title, setTitle] = useState(children)
```